packages/expo-updates/android/src/main/java/expo/modules/updates/db/entity/AssetEntity.kt

```diff
@@ -68,4 +68,10 @@ class AssetEntity(@field:ColumnInfo(name = "key") var key: String?, var type: St
 
   @Ignore
   var scales: Array<Float>? = null
+
+  internal fun getFileExtension(): String {
+    return this.type?.let {
+      if (it.startsWith(".")) it else ".$it"
+    } ?: ""
```

Comment:
```suggestion
    val type = this.type ?: return ""
    return if (type.startsWith(".")) {
      type
    } else {
      ".$it"
    }
```
I think it's more readable, but it's up to you